selfdrive/car/chrysler/interface.py

```diff
@@ -62,13 +62,17 @@ def _get_params(ret, candidate, fingerprint, car_fw, experimental_long, docs):
       ret.wheelbase = 3.88
       ret.steerRatio = 16.3
       ret.mass = 2493. + STD_CARGO_KG
-      ret.minSteerSpeed = 14.5
-      # Older EPS FW allow steer to zero
-      if any(fw.ecu == 'eps' and fw.fwVersion[:4] <= b"6831" for fw in car_fw):
-        ret.minSteerSpeed = 0.
+      CarInterfaceBase.configure_torque_tune(candidate, ret.lateralTuning)
+      ret.minSteerSpeed = 0.5
+      ret.minEnableSpeed = 14.6
```

Comment:
```suggestion
      ret.minEnableSpeed = 14.5
```
Is the additional 0.1 required?